app/js/views/services/light.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'components/react';
+
+import NavigationMenu from 'js/views/navigation-menu';
+import TagList from 'js/views/tag-list';
+
+export default class LightService extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: props.service.tags,
+      tags: []
+    };
+
+    this.foxbox = props.foxbox;
+    this.service = props.service;
+
+    this.onServiceStateChanged = this.onServiceStateChanged.bind(this);
+  }
+
+  componentDidMount() {
+    this.populateTags();
+
+    this.foxbox.addEventListener(
+      'service-state-change', this.onServiceStateChanged
+    );
+  }
+
+  componentWillUnmount() {
+    this.foxbox.removeEventListener(
+      'service-state-change', this.onServiceStateChanged
+    );
+  }
+
+  onServiceStateChanged(service) {
+    if (service.id !== this.service.id) {
+      return;
+    }
+
+    this.service = service;
+    this.setState({ data: service.tags });
+  }
+
+  populateTags() {
```

Comment:
It's the fake tags for now, purely frontend. I'll keep them for now but they'll probably go away soon.